postcss.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  plugins: [
+    require('autoprefixer')({
```

Comment:
I've tried it on recent versions of FF, Chrome, and Safari, but not certain about IE or older browser versions.